src/main/scala/org/allenai/wikitables/WikiTablesSemanticParserCli.scala

```diff
@@ -326,12 +328,12 @@ class WikiTablesSemanticParserCli extends AbstractCli() {
           
       val beam = results.executions.slice(0, 10)
       val correct = beam.map { x =>
-        val simplified = simplifier.apply(x.value.decodeExpression)
```

Comment:
Why did you get rid of the simplifier? The simplifier just makes logical forms easier to read. Its output is semantically equivalent to its input.